### DISCUSSION TITLE
Any plans on supporting Cargo (Rust) packages ?

### DISCUSSION THREAD

**Discussion by endersonmaia** [community/community#11926] [Packages]
Are there any plans to support [Cargo](https://doc.rust-lang.org/stable/cargo/reference/registries.html) (Rust) packages within GitHub Packages ?

Or is there a public roadmap for additional types of packages to be supported by GttHub Packages in the future ?

**Comment by RobbieMcKinstry** [comment]
I can't understate how impactful this would be for me. As far as I'm aware, there really aren't any great crates.io hosts out there (with a free tier). Being able to store my private crates on GitHub Packages would really be a game-changer for me.

**Comment by beingminimal** [comment]
@otter-computer any update or consideration for this?
